### ISSUE TITLE
debuffs: Duration with "Improved Shadow Word: Pain" is not calculated correct

### ISSUE THREAD

**Issue by Grookshank** [shagu/pfUI#624]
If you spec a priest into "Improved Shadow Word: Pain" the debuff on your target should be 3/6 sec. longer. Currently this is not working and the talent is not calculated.

**Comment by shagu** [comment]
Quote from https://github.com/shagu/pfUI/issues/627
> This is related on how a debuff timers is working for Vanilla. Let me try to explain this:
> The Vanilla Wow API doesn't provide any function that enables me to read any remaining debuff times for enemies. So the addon has to scan the combat log for messages that indicate whenever a debuff has been applied to an enemy and then query its internal spell database (https://github.com/shagu/pfUI/blob/master/env/locales_enUS.lua#L1445) to retrieve the default time that debuff usually has. There aren't any UUIDs and the combat log only contains the name and the spell.
> tl;dr: It's impossible for me to distinguish between two mobs if they share the same name.

Same reason here, as I might be able to extend this for your own spells, it will remain wrong for every other priest, because you can't read his talents. Extending the debuff timers to be more accurate is a never ending story and will consume more and more time. I decided to stop at the break even point of functionality and effort.

**Comment by stva23** [comment]
As a workaround you can adjust the debuff time in your locale-file (e.g.  env/locales_enUS.lua) from 18 to 24. Works for me. Would be nice if this could be changed in the settings dialog.
